src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -1388,7 +1388,7 @@ internal static int FromHex(char digit)
                     : ((int)digit - (int)'a'))
                     + 10);
             }
-            throw new ArgumentException(nameof(digit));
+            throw new ArgumentException(SR.Argument_NotHexDigit, nameof(digit));
```

Comment:
Rather than adding a resource string here, seems like this should just be an ArgumentOutOfRangeException, but @davidsh can comment on this and the other Uri changes.